cmd/influxd/upgrade/upgrade.go

```diff
@@ -453,7 +455,10 @@ func runUpgradeE(*cobra.Command, []string) error {
 		)
 	}
 
-	log.Info("Upgrade successfully completed. Start service now")
+	log.Info(
+		"Upgrade successfully completed. Start service now to log in",
+		zap.String("login_url", options.source.dbURL),
```

Comment:
@timhallinflux this ends up logging lines like:
```
{"level":"info","ts":1607373509.091895,"caller":"upgrade/upgrade.go:458","msg":"Upgrade successfully completed. Start service now to log in","login_url":"http://localhost:8086"}
```
Look good to you?

Reply:
+1

Reply:
"Start service now to login" sounds to me like going to http://localhost:8086 will start the service for me - for a user that just ran the upgrade this is not the case.

Maybe `Upgrade successfully completed. Ensure influxd service is started and then log in","login_url":"http://localhost:8086"}` or similar would be clearer?

Reply:
"Ensure influxd service is started" sounds to me like we expect `upgrade` to start the service as its last step. How does the latest push look?